pandas/io/tests/test_common.py

```diff
@@ -90,6 +91,18 @@ def test_iterator(self):
         expected.index = [0 for i in range(len(expected))]
         tm.assert_frame_equal(concat(it), expected.iloc[1:])
 
+    def test_temporary_file(self):
+        # GH13398
+        from tempfile import TemporaryFile
+        new_file = TemporaryFile("w+")
+        new_file.write(self.data2)
+        new_file.flush()
+        new_file.seek(0)
+
+        result = read_csv(new_file, sep=r"\s+", engine="python")
```

Comment:
is there a corresponding test for the c engine that does the same? or is this specifically an issue with the python engine?